airbyte-integrations/connectors/source-s3/pyproject.toml

```
@@ -27,7 +27,7 @@ source-s3 = "source_s3.run:run"
 
 [tool.poetry.dependencies.airbyte-cdk]
 extras = [ "file-based",]
-version = "==0.61.0"
+version = "==0.63.1"
```

@clnoll can we relax the pinning?
```suggestion
version = "^0"
```

I'd mean that a `poetry upgrade airbyte-cdk` will download the latest CDK. But if no manual upgrade happens the version defined in `poetry.lock` will continue to be used.

---

Sure, that works.